example/cycle/test/test.js

```diff
@@ -7,31 +7,25 @@ import * as actions from '../../actions';
 
 import { fetchReposByUser, searchUsers } from '../';
 
-function assertSourcesSinks(sources, sinks, main, done) {
-  const Time = mockTimeSource();
+function assertSourcesSinks(sources, sinks, main, done, timeOpts = {}) {
```

Comment:
Actually this won't work because it doesn't take into account the source method arguments. You could have a main with multiple `DOM.select('...')`, and this will pass the same stream to both. Same for HTTP. I wonder if we can build an API that allows users to specify the methods to mock (HTTP.select(), DOM.select().events(), or other weird ones), while also allowing arguments